privacyidea/api/lib/policyhelper.py

```diff
@@ -0,0 +1,117 @@
+# -*- coding: utf-8 -*-
+#
+#  2023-01-27 Cornelius Kölbel <cornelius@privacyidea.org>
+#             Create this module for enabling decorators for API calls
+#
+#  License:  AGPLv3
+#  contact:  http://www.privacyidea.org
+#
+# This code is free software; you can redistribute it and/or
+# modify it under the terms of the GNU AFFERO GENERAL PUBLIC LICENSE
+# License as published by the Free Software Foundation; either
+# version 3 of the License, or any later version.
+#
+# This code is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU AFFERO GENERAL PUBLIC LICENSE for more details.
+#
+# You should have received a copy of the GNU Affero General Public
+# License along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+"""
+These are logical policy functions that are usually used in policy API decorators, but
+in some cases also used beside the API.
+Like policies, that are supposed to read and pass parameters during enrollment of a token.
+"""
+
+import logging
+from privacyidea.lib.log import log_with
+from privacyidea.lib.policy import Match, SCOPE, ACTION
+from privacyidea.lib.error import PolicyError
+
+log = logging.getLogger(__name__)
+
+
+@log_with(log)
+def get_init_tokenlabel_parameters(g, params=None, token_type="hotp", user_object=None):
```

Comment:
since this is not only for the tokenlabel, we should rename the function to something like `get_init_authenticator_paramsters()`

Reply:
In a perfect world you are right.

I chose this name, since the decorator is actually called "init_tokenlabel". So I wanted to match the actual decorator - even if the naming of the decorator is also not perfect.